### PR TITLE
adding SimplyCyber to creators list.

### PR DESCRIPTION
**Pull request by gerryguy311**
per our email communication. adding simply cyber using the same format as other creators in the JSON.

### CODE REVIEW

File: src/assets/data/creators.json [hunk 1]
```diff
@@ -1,4 +1,18 @@
 [
+ {
+   "name": "SimplyCyber",
+   "location": "South Carolina, USA",
+   "timezone": "EST (UTC-5)",
+   "avatar": "Gerald_Auger_SimplyCyber.png",
+   "schedule": "New Videos Every Monday at Noon EST Helping YOU Make and Take a Cybersecurity Care$
```
Comment (ZetaTwo) [inline src/assets/data/creators.json:7]: This seems to have been cut off.

File: src/assets/data/creators.json [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+   "tags": ["Career", "Cybersecurity Awareness", "RED/BLUE TEAM", "Breaking into industry", "Mento$
```
Comment (ZetaTwo) [inline src/assets/data/creators.json:8]: This seems to have been cut off.

File: src/assets/data/creators.json (hunk 1 above)
Comment (ZetaTwo) [reply src/assets/data/creators.json:7]: Also, please keep it short.

**Review by ZetaTwo — CHANGES_REQUESTED** [review]
See comments

File: src/assets/data/creators.json (hunk 1 above)
Comment (gerryguy311) [reply src/assets/data/creators.json:7]: "schedule": "Monday 12:00 EST. Make and Take a Cyber Career Further, Faster"

File: src/assets/data/creators.json (hunk 2 above)
Comment (gerryguy311) [reply src/assets/data/creators.json:8]: "tags": ["Career", "Cybersecurity Awareness", "Soc Analyst", "Mentoring"]

**Comment by gerryguy311** [comment]
Provided replies to two text fields that were cutoff. Thanks again for the opportunity.

**Pull request merged**